src/onboarding/registration/NameAndPicture.test.tsx

```diff
@@ -65,4 +66,82 @@ describe('NameAndPictureScreen', () => {
     )
     expect(queryByTestId('PictureInput')).toBeNull()
   })
+
+  it('render header title correctly when createAccountCopyTestConfig is "control"', () => {
+    const store = createMockStore({
+      account: {
+        choseToRestoreAccount: false,
+      },
+      app: {
+        createAccountCopyTestConfig: 'control',
+      },
+    })
+
+    let headerTitle: React.ReactNode
+    ;(mockNavigation.setOptions as jest.Mock).mockImplementation((options) => {
+      headerTitle = options.headerTitle()
+    })
+
+    render(
+      <Provider store={store}>
+        <NameAndPicture {...mockScreenProps} />
+      </Provider>
+    )
+
+    const { queryByText } = render(<Provider store={store}>{headerTitle}</Provider>)
+
+    expect(queryByText('createAccount')).toBeTruthy()
```

Comment:
minor but if we expect something to be there, generally using the `getBy*` selectors are better because they throw more meaningful errors when the element is not found. generally i use `queryBy*` only when asserting that things shouldn't be there

Reply:
Thanks for the explanation! I changed it to getBy*